src/main/java/Archibald.java

```diff
@@ -0,0 +1,104 @@
+import java.util.Scanner;
+
+public class Archibald {
+
+    private static Task[] tasks = new Task[100];
+    private static int taskCount = 0;
+
+    public static void printArchibaldResponse(String message) {
+        System.out.println("____________________________________________________________");
+        System.out.println(message);
+        System.out.println("____________________________________________________________");
+    }
+
+    public static void addTask(String input) {
+        if (taskCount >= tasks.length) {
+            printArchibaldResponse("Error: Task list is full!");
+            return;
+        }
+
+        String[] parts = input.split(" ", 2);
+        String type = parts[0].toLowerCase();
+        String details = parts.length > 1 ? parts[1] : "";
+
+        switch (type) {
+            case "todo":
+                tasks[taskCount++] = new Todo(details);
+                break;
+            case "deadline":
+                String[] deadlineParts = details.split(" /by ", 2);
+                tasks[taskCount++] = new Deadline(deadlineParts[0], deadlineParts[1]);
+                break;
+            case "event":
+                String[] eventParts = details.split(" /from | /to ");
```

Comment:
The split logic assumes that the /from and /to keywords will always be formatted correctly. If the user doesn't follow this format exactly, it may lead to an ArrayIndexOutOfBoundsException.